apps/examples/testcase/le_tc/kernel/tc_libc_stdlib.c

```diff
@@ -668,7 +668,9 @@ static void tc_libc_stdlib_div(void)
  ****************************************************************************/
 int libc_stdlib_main(void)
 {
+#ifndef CONFIG_DISABLE_PTHREAD
 	tc_libc_stdlib_abort();
```

Comment:
The abort is working on task as well. Why don't you change creating task instead of pthread?